.travis.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         - ./gradlew compileDebugUnitTestSources
         - ./gradlew compileReleaseSources
     - stage: Testing (Parallel integration & unit tests) 🏎
-      script: ./testing integration_tests
+      script: travis_wait 30 ./testing integration_tests
```

Comment:
Why are you having travis waiat 30s before running the integration tests?

Reply:
probably making sure that the emulator is ready. We had issues with the timeout of the emulator

Reply:
All gucci.